### DISCUSSION TITLE
Feedback for improvements on our Web Translation Tool

### DISCUSSION THREAD

**Discussion by TaylorHo** [hotaydev/dnd-beyond-kit#16] [Ideas]
Give Ideas, suggestions, and useful feedback about our [Web Translation Tool](https://dnd-beyond-kit.hotay.dev/translate).

All feedback and suggestions are appreciated!

**Comment by Disispower** [comment]
Hi, i could possible help with translating D&D beyond to the dutch language. If you are interested.

**Reply by Disispower** [reply]
Since i am a native Dutch speaker, who can also speak and read English reasonable well

**Reply by TaylorHo** [reply]
Hi @Disispower, that would be awesome!

I'll create the files for the new language, leaving the content initially in English. After that, I'll ask you in the comments (one comment on each line of code) about the translation. Take your time to answer and, when ready, we'll ship the new language in the extension ☺️

**Reply by TaylorHo** [reply]
@Disispower I have placed some comments asking for translations in [this link on GitHub](https://github.com/hotaydev/dnd-beyond-kit/pull/22/commits/5ac682358439baaa9d502d82900eb4fa9473bfbb).

You can answer directly in each comment in the link and I'll change the content in the code after it.

Thank you :D

**Reply by TaylorHo** [reply]
@Disispower Thank you so much for your contributions to these translations!
The full UI now has Dutch translations, which will be made available in the next version sent to the Extension's stores.

Also, now the [translation tool](https://dnd-beyond-kit.hotay.dev/translate) has a place for you to translate the D&D Beyond interface into Dutch. Please, check the page, change the "Language to translate" to Dutch, and start translating 😄
